### ISSUE TITLE
Incorrect results when opening cursor IDBKeyRange with negative numbers 

### ISSUE THREAD

**Issue by patjackson52** [indexeddbshim/IndexedDBShim#62]
Appears to be a bug when opening a cursor with a IDBKeyRange that contains negative numbers when using WebSQL.  I am attempting a geospatial query by using lat/ long indexes.  This works fine under indexeddb in Chrome, but when I run in safari I only get results for lat(which is all positive with my data).  The longitude range query returns 0 results.  Partial code below:

```
var latIndex = objectStore.index("lat");
var longiIndex=objectStore.index("longi");
var r = 5000; // require distance
lat_range = IDBKeyRange.bound(lat - r, lat + r);
longi_range=IDBKeyRange.bound(longi-r,longi+r);

var latIndexCursorReq = latIndex.openCursor(lat_range);
var latIndexes=[];
var longiIndexes=[];

latIndexCursorReq.onsuccess = function(){
var cursor = latIndexCursorReq.result;

if(cursor){
     latIndexes.push(cursor.primaryKey);
     cursor['continue']();           
}else{  
console.log(true, "Lat Cursor Iteration completed");
var longiIndexCursorReq = longiIndex.openCursor(longi_range);
    }
```

**Comment by axemclion** [comment]
Thnx for the bug report, will look into it.

**Comment by JamesMessinger** [comment]
This should be fixed now (as of version 1.0.5).  Please give it a try and let us know if you find any problems.  Thanks!
